examples/hybrid/bubble_3d.jl

```diff
@@ -0,0 +1,284 @@
+        top = Operators.Extrapolate(),
+    )
+
+    uₕ = @. Yc.ρuₕ / Yc.ρ
+    w = @. ρw / If(Yc.ρ)
+    wc = @. Ic(ρw) / Yc.ρ
+    p = @. pressure(Yc.ρθ)
+    θ = @. Yc.ρθ / Yc.ρ
+    Yfρ = @. If(Yc.ρ)
+
+    ### HYPERVISCOSITY
+    # 1) compute hyperviscosity coefficients 
+    @. dYc.ρθ = hdiv(hgrad(θ))
+    @. dYc.ρuₕ = hdiv(hgrad(uₕ))
+    @. dρw = hdiv(hgrad(w))
+    Spaces.weighted_dss!(dYc)
+    Spaces.weighted_dss!(dρw)
+
+    κ₄ = 100.0 # m^4/s
+    @. dYc.ρθ = -κ₄ * hdiv(Yc.ρ * hgrad(dYc.ρθ))
+    @. dYc.ρuₕ = -κ₄ * hdiv(Yc.ρ * hgrad(dYc.ρuₕ))
+    @. dρw = -κ₄ * hdiv(Yfρ * hgrad(dρw))
+
+    # density
+    @. dYc.ρ = -∂(ρw)
+    @. dYc.ρ -= hdiv(Yc.ρuₕ)
+
+    # potential temperature
+    @. dYc.ρθ += -(∂(ρw * If(Yc.ρθ / Yc.ρ)))
+    @. dYc.ρθ -= hdiv(uₕ * Yc.ρθ)
+
+    # Horizontal momentum
+    @. dYc.ρuₕ += -uvdivf2c(ρw ⊗ If(uₕ))
+    Ih = Ref(
+        Geometry.Axis2Tensor(
+            (Geometry.UVAxis(), Geometry.UVAxis()),
+            @SMatrix [1.0 0.0; 0.0 1.0]
+        ),
+    )
+    @. dYc.ρuₕ -= hdiv(Yc.ρuₕ ⊗ uₕ + p * Ih)
+    # -------------------> 
+
+    # vertical momentum
+    @. dρw +=
+        B(
+            Geometry.transform(
+                Geometry.WAxis(),
+                -(∂f(p)) - If(Yc.ρ) * ∂f(Φ(coords.z)),
+            ) - vvdivc2f(Ic(ρw ⊗ w)),
+        ) + fcf(wc, ρw)
+    uₕf = @. If(Yc.ρuₕ / Yc.ρ) # requires boundary conditions
+    @. dρw -= hdiv(uₕf ⊗ ρw)
+
+    ### UPWIND FLUX CORRECTION 
+    upwind_correction = true
+    if upwind_correction
+        @. dYc.ρ += fcc(w, Yc.ρ)
+        @. dYc.ρθ += fcc(w, Yc.ρθ)
+        @. dYc.ρuₕ += fcc(w, Yc.ρuₕ)
+        @. dρw += fcf(wc, ρw)
+    end
+
+    ### DIFFUSION
+    κ₂ = 0.0 # m^2/s
+    #  1a) horizontal div of horizontal grad of horiz momentun
+    @. dYc.ρuₕ += hdiv(κ₂ * (Yc.ρ * hgrad(Yc.ρuₕ / Yc.ρ)))
+    #  1b) vertical div of vertical grad of horiz momentun
+    @. dYc.ρuₕ += uvdivf2c(κ₂ * (Yfρ * ∂f(Yc.ρuₕ / Yc.ρ)))
+
+    #  1c) horizontal div of horizontal grad of vert momentum
+    @. dρw += hdiv(κ₂ * (Yfρ * hgrad(ρw / Yfρ)))
+    #  1d) vertical div of vertical grad of vert momentun
+    @. dρw += vvdivc2f(κ₂ * (Yc.ρ * ∂c(ρw / Yfρ)))
+
+    #  2a) horizontal div of horizontal grad of potential temperature
+    @. dYc.ρθ += hdiv(κ₂ * (Yc.ρ * hgrad(Yc.ρθ / Yc.ρ)))
+    #  2b) vertical div of vertial grad of potential temperature
+    @. dYc.ρθ += ∂(κ₂ * (Yfρ * ∂f(Yc.ρθ / Yc.ρ)))
+
+    Spaces.weighted_dss!(dYc)
+    Spaces.weighted_dss!(dρw)
+    return dY
+end
+
+dYdt = similar(Y);
+rhs!(dYdt, Y, nothing, 0.0);
+
+
+# run!
+using OrdinaryDiffEq
+Δt = 0.05
+prob = ODEProblem(rhs!, Y, (0.0, 1.0))
+sol = solve(
+    prob,
+    SSPRK33(),
+    dt = Δt,
+    saveat = 1.0,
+    progress = true,
+    progress_message = (dt, u, p, t) -> t,
+);
+
+ENV["GKSwstype"] = "nul"
+#import Plots
+#Plots.GRBackend()
+
+#dirname = "bubble_3d"
+#path = joinpath(@__DIR__, "output", dirname)
+#mkpath(path)
+#TODO: Slab plots + VTK output support for 3D domains
```

Comment:
Are these stale comments? Or meant to be decommented?

Reply:
we don't have visualization support for 3D extruded fields